### DISCUSSION TITLE
Manual entery for paring information

### DISCUSSION THREAD

**Discussion by pschute** [maxileith/homebridge-appletv-enhanced#69] [Q&A]
Is there a way to manually input the airplay credentials & companion credentials instead of it searching for the ATV on my network? I am able to use PYATV to collect the manually and was wondering if there is a verbal some where that I can just input them. 

Thanks!

**Comment by maxileith** [comment]
Hi @pschute ,

you can input the credentials of your Apple TV manually by pasting them in a file. Go to your homebridge storage path, then `appletv-enhanced/AABBCCDDEEFF/credentials.txt` will be your file to paste the credentials to. `AABBCCDDEEFF` should be adjusted to the MAC-Address of your Apple TV.

However, even when adding credentials manually, the plugin still needs to discover your Apple TV on itself.

Could you please look into Issue #20 and check if this is your problem? Please let me know if it is. I am currently waiting for a PR in a dependency to get merged before I can continue with the fix.

**Reply by pschute** [reply]
Yea, I seem to be having the same issue that is outlines in #20

**Reply by maxileith** [reply]
Okay, than I would ask you to be patient. Once the PR mentioned in this issue is merged in the pyatv project, I will continue with the fix ...
